conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -116,45 +137,55 @@ def truncate_content(s):
            dic['records_count'] = no_records
        return dics
 
+def is_in_range(pid):
+    for lr, ur in ingest_range:
+        if pid >= lr and pid <= ur:
+           return True
+    return False
+
+def is_time_range(ts):
+    dt = datetime.strptime(ts, "%Y-%m-%d %H:%M:%S.%f %Z")
+    if not(dt.year == 2017): return (dt.year < 2017) 
+    else:
+       return (dt.month < 6)
 
 class WriteDecompressedFile(beam.DoFn):
   def process(self, element):
-    logging.info('USERLOG: Working on %s' % element)
+    """
+      Process Mode:
+    """
     chunk_name = element
 
-    in_file_path = path.join('gs://wikidetox-viz-dataflow/raw-downloads/', chunk_name)
+    in_file_path = path.join('gs://wikidetox-viz-dataflow/raw-downloads-20180201/20180201-dumps/', chunk_name)
 
     logging.info('USERLOG: Running gsutil %s ./' % in_file_path)
     cp_local_cmd = (['gsutil', 'cp', in_file_path, './'])
     subprocess.call(cp_local_cmd)
 
     logging.info('USERLOG: Running ingestion process on %s' % chunk_name)
     ingestion_cmd = ['python2', '-m', 'ingest_utils.run_ingester', '-i', chunk_name]
-    kill = lambda process, status: process.kill();status='timeout'
-    status = 'success'
     ingest_proc = subprocess.Popen(ingestion_cmd, stdout=subprocess.PIPE, bufsize = 4096)
-#    timer = Timer(my_timeout, kill, (ingest_proc, status))
-#    timer.start()
     cnt = 0
     maxsize = 0
     last_revision = 'None'
+    last_completed = time.time()
     for i, line in enumerate(ingest_proc.stdout):
       try:
-         content = json.loads(line) 
+          content = json.loads(line) 
       except:
          break
       last_revision = content['rev_id']
       ret = truncate_content(line)
```

Comment:
Do you still want/need to truncate here? These now go to file, so I don't think truncating is needed anymore... right?

Reply:
Ah yes no truncating needed.